c_reference/src/conv1d.c

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#include<stdlib.h>
+#include<math.h>
+
+#include"conv1d.h"
+#include"conv_utils.h"
+#include"utils.h"
+
+int conv1d_lr(float *output_signal, unsigned out_T, unsigned out_channels, const float *input_signal,
+  unsigned in_T, unsigned in_channels, int padding, unsigned kernel_size,
+  const void* params, int activations) {
+
+  const ConvLayers_LR_Params* tparams= (ConvLayers_LR_Params*) params;
+
+  if (padding == -1) {
```

Comment:
the header files doesn't say much about what setting padding to -1 (or any other value) means

Reply:
I added some documentation for padding. I removed this -1 part altogether to avoid confusions

Reply:
what was the -1 for?

Reply:
I originally thought of keeping a case to calculate the padding automatically to maintain the time steps. 

So if it was -1, the padding would be calculated such that input time steps would be equal to the output time steps. It was similar to the "same" convolution in tensorflow

But since it would be confusing for the user, I removed those lines